store/cassandra/config.go

```diff
@@ -67,7 +68,7 @@ func ConfigSetup() *flag.FlagSet {
 	cas.StringVar(&CliConfig.Keyspace, "keyspace", CliConfig.Keyspace, "cassandra keyspace to use for storing the metric data table")
 	cas.StringVar(&CliConfig.Consistency, "consistency", CliConfig.Consistency, "write consistency (any|one|two|three|quorum|all|local_quorum|each_quorum|local_one")
 	cas.StringVar(&CliConfig.HostSelectionPolicy, "host-selection-policy", CliConfig.HostSelectionPolicy, "")
-	cas.IntVar(&CliConfig.Timeout, "timeout", CliConfig.Timeout, "cassandra timeout in milliseconds")
+	cas.DurationVar(&CliConfig.Timeout, "timeout", CliConfig.Timeout, "cassandra timeout")
```

Comment:
This will cause problems on existing MT deployments.  The current config setting uses milliseconds, but when parsed as a duration they will now be treated as nanoseconds.

We need to ensure that doesnt happen

Reply:
Lets combine those ideas.
- make it a stringVar
- if the value is just a number, treat it as ms, but log a warning that using a millisecond int is depreciated.
- otherwise parse it as a duration.

Reply:
+1

Reply:
@woodsaj @Dieterbe done, please take a look now